pkg/kudoctl/packages/resolver/resolver.go

```diff
@@ -42,8 +44,16 @@ func New(repo *repo.Client) *PackageResolver {
 func (m *PackageResolver) Resolve(name string, appVersion string, operatorVersion string) (p *packages.Package, err error) {
 
 	// Local files/folder have priority
-	if _, err = m.local.fs.Stat(name); err == nil {
-		clog.V(2).Printf("local operator discovered: %v", name)
+	_, err = m.local.fs.Stat(name)
+	// force local operators usage to be either absolute or express a relative path
+	// or put another way, a name can NOT be mistaken to be the name of a local folder
+	if filepath.IsAbs(name) || (filepath.Base(name) != name && err == nil) {
```

Comment:
I'm not sure I understand the logic: on the left, we call the `m.local.Resolve` if the file/folder actually *exists*. Here, you could have an absolute but *non-existing* path and still call the `Resolve` method.